pkg/sqlite/deprecate_test.go

```diff
@@ -0,0 +1,330 @@
+package sqlite
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/blang/semver/v4"
+	"github.com/operator-framework/api/pkg/lib/version"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-registry/pkg/api"
+	image2 "github.com/operator-framework/operator-registry/pkg/image"
```

Comment:
Minor nit, but any reason why we need to rename this import declaration and can't just use the regular path?